cmd/shell.go

```diff
@@ -130,18 +129,18 @@ func (log promLogger) Println(args ...interface{}) {
 	log.AuditErr(fmt.Sprintln(args...))
 }
 
-// StatsAndLogging constructs a metrics.Scope and an AuditLogger based on its config
-// parameters, and return them both. It also spawns off an HTTP server on the
-// provided port to report the stats and provide pprof profiling handlers.
-// Crashes if any setup fails.
+// StatsAndLogging constructs a prometheus registerer and an AuditLogger based
+// on its config parameters, and return them both. It also spawns off an HTTP
+// server on the provided port to report the stats and provide pprof profiling
+// handlers. NewLogger and newStatsRegistry will call os.Exit on errors.
 // Also sets the constructed AuditLogger as the default logger, and configures
 // the cfssl, mysql, and grpc packages to use our logger.
 // This must be called before any gRPC code is called, because gRPC's SetLogger
 // doesn't use any locking.
-func StatsAndLogging(logConf SyslogConfig, addr string) (metrics.Scope, blog.Logger) {
+func StatsAndLogging(logConf SyslogConfig, addr string) (prometheus.Registerer, blog.Logger) {
 	logger := NewLogger(logConf)
-	scope := newScope(addr, logger)
-	return scope, logger
+	stats := newStatsRegistry(addr, logger)
+	return stats, logger
```

Comment:
minor nit: I think the intermediate vars can be replaced with a straight-through `return`.
```suggestion
	return newStatsRegistry(addr, logger), NewLogger(logConf)
```

Reply:
It can be done for `newStatsRegistry`, but since it ref's `logger` then `NewLogger` needs to be done in the func body 😞.

Reply:
Ahhh, good point. Sorry I missed that!